cookbooks/global_melt.prm

```diff
@@ -29,14 +29,14 @@ set Number of cheap Stokes solver steps    = 100
 # time scale than the flow of melt, so we want to decouple the advection
 # of melt (and temperature) and the melting process itself. To do that, 
 # we use the operator splitting scheme, and define that for every 
-# advection time step, we want to do at least 50 reaction time step, 
-# and in addition, we limit the maximum reaction time step size to 
-# 500,000 years. 
+# advection time step, we want to do at least at least 10 reaction time steps, 
```

Comment:
"at least at least" -> "at least"

Reply:
Also, it might be cleaner to breakup this sentence. The second sentence could read "If the advection time step is larger than 10,000 years, one should do more reaction time steps".

In the sentence above, I am assuming the user would need to adjust the number of reaction time steps if the advection time step is larger than 10,000 years.